Documentation/contributor-guide/triage_issues.md

```diff
@@ -4,43 +4,107 @@
+
+If the problem can not be reproduced or requires more information, leave a comment for the issue reporter as soon as possible while the issue will be fresh for the issue reporter.
+
+### Feature requests
+
+New feature requests should be created via the etcd feature request template and in theory already have the `type/feature` label, however if this is missing for an issue you determine to be a feature please add the label manually.
+
+### Test flakes
+
+Test flakes are a specific type of bug that the etcd project tracks seperately as these are a priority to address. These should be created via the test flake template and in theory already have the `type/flake` label, however if this is missing for an issue you determine to be related to a flaking test please add the label manually.
+
+## Step 4 - Define the areas impacted
 
-## Validate if an issue is a bug
+Adding an `area` label to an issue helps create visibility on which areas of the etcd project require attention and helps contributors find issues to work on relating to their particular skills or knowledge of the etcd codebase.
 
-Validate if the issue is indeed a bug. If not, add a comment with findings and close trivial issue. For non-trivial issue, wait to hear back from issue reporter and see if there is any objection. If issue reporter does not reply in 30 days, close the issue. If the problem can not be reproduced or require more information, leave a comment for the issue reporter.
+If an issue crosses multiple domains please add additional `area` labels to reflect that.
 
-## Inactive issues
+Below is a brief summary of the area labels in use by the etcd project along with any comments on their use:
 
-Issues that lack enough information from the issue reporter should be closed if issue reporter do not provide information in 60 days.
+| Label                   | Comments                                                      |
+| ---                     | ---                                                           |
+| area/community          |                                                               |
+| area/raft               |                                                               |
+| area/clientv3           |                                                               |
+| area/performance        |                                                               |
+| area/security           |                                                               |
+| area/tls                |                                                               |
+| area/auth               |                                                               |
+| area/etcdctl            |                                                               |
+| area/etcdutl            |                                                               |
```

Comment:
This `area/etcdutl` label doesn't currently exist but I think should. Can a maintainer please create it if they agree?

Reply:
SG

Reply:
Agreed & thx & created